src/ciadmin/generate/worker_pools.py

```diff
@@ -201,109 +201,77 @@ def get_azure_provider_config(
             "wstServerURL", azure_config["wst_server_url"]
         )
     tags = config.get("tags", {})
-    # Temporary conditional language while we move pools over to the newer
-    # method that is in the else clause
-    # Refrence https://mozilla-hub.atlassian.net/browse/RELOPS-212
-    if config.get("old-style-worker-config"):
-        launch_configs = []
-        for location in locations:
-            for vmSize in vmSizes:
-                loc = location.replace("-", "")
-                ImageId = image.image_id(provider_id, location)
+
+    launch_configs = []
+    for location in locations:
+        for vmSize in vmSizes:
+            loc = location.replace("-", "")
+            DeploymentId = image.image_id(provider_id, "deployment_id")
+            try:
+                manifest = image.image_id(provider_id, "manifest")
+            except:
+                pass
+            try:
+                version = image.image_id(provider_id, "version")
+            except:
+                pass
+            if provider_id == "azure_trusted":
+                subscription_id = azure_config["trusted_subscription"]
+            else:
                 subscription_id = azure_config["untrusted_subscription"]
-                subscription_id = f"/subscriptions/{subscription_id}"
-                resource_suffix = f"{location}-{purpose}"
-                rgroup = f"rg-{resource_suffix}"
-                vnet = f"vn-{resource_suffix}"
-                snet = f"sn-{resource_suffix}"
-                subnetId = (
-                    f"{subscription_id}/resourceGroups/{rgroup}/providers/"
-                    f"Microsoft.Network/virtualNetworks/{vnet}/subnets/{snet}"
-                )
+            subscription_id = f"/subscriptions/{subscription_id}"
+            resource_suffix = f"{location}-{purpose}"
+            rgroup = f"rg-{resource_suffix}"
+            vnet = f"vn-{resource_suffix}"
+            snet = f"sn-{resource_suffix}"
+            subnetId = (
+                f"{subscription_id}/resourceGroups/{rgroup}/providers/"
+                f"Microsoft.Network/virtualNetworks/{vnet}/subnets/{snet}"
+            )
+            if "version" in locals() and version:
```

Comment:
Oh I see, you're checking that it's defined here. Probably still better to just set it to `None` in the `except` clause and then change this to `if version:`.